### PR TITLE
fix: trim quote characters in osinfo

### PR DESCRIPTION
**Pull request by NDStrahilevitz**
Some distributions may use quotes in their os-info file (ex: centos). The osinfo helpers must handle these cases for correct parsing.

### CODE REVIEW

File: helpers/osinfo.go [hunk 1]
```diff
@@ -226,7 +226,9 @@ func (btfi *OSInfo) discoverOSDistro() error {
 		}
 		btfi.osReleaseFieldValues[keyID] = val[1]
 		if keyID == OS_ID {
-			btfi.osReleaseID = stringToOSReleaseID[strings.ToLower(val[1])]
+			osVal := strings.ToLower(val[1])
+			osVal = strings.Trim(osVal, "\"") // trim potential parentheses (in centos)
```
Comment (geyslan) [inline helpers/osinfo.go:230]: quotes?

Comment (NDStrahilevitz) [reply helpers/osinfo.go:230]: ?

**Comment by NDStrahilevitz** [comment]
@geyslan Are self test failures intermittent? As I haven't changed anything related as far as I can tell.

File: helpers/osinfo.go (hunk 1 above)
Comment (geyslan) [reply helpers/osinfo.go:230]: image

Comment (NDStrahilevitz) [reply helpers/osinfo.go:230]: You mean the comment should be
`// trim potential quotes (in centos`?

Comment (NDStrahilevitz) [reply helpers/osinfo.go]: Done.

**Comment by geyslan** [comment]
> @geyslan Are self test failures intermittent? As I haven't changed anything related as far as I can tell.

Possibly related to timeout, right? I've increased it in https://github.com/aquasecurity/libbpfgo/pull/343 (waiting review).

**Review by geyslan — APPROVED** [review]
LGTM

**Pull request merged**